### DISCUSSION TITLE
hasOne must return array

### DISCUSSION THREAD

**Discussion by ZsarkZ** [laravel/nova-issues#4590] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4
- PHP Version: 8
- Database Driver & Version:
- Operating System and Version: ubuntu 22.04
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

I have error with Flexible field and in this issue
[https://github.com/whitecube/nova-flexible-content/issues/360](https://github.com/whitecube/nova-flexible-content/issues/360)
and  me told that hasOne nova field wrong work, can you help me?

**Comment by crynobone** [comment, marked as answer]
`HasOne` field has the correct structure and it is working as expected. It could just be the field isn't compatible with `nova-flexible-content` package.

**Reply by ZsarkZ** [reply]
thanks
